pkg/lvm/lvm_util.go

```diff
@@ -891,12 +926,20 @@ func ListLVMPhysicalVolume() ([]PhysicalVolume, error) {
 		return nil, err
 	}
 
+	cmd := PVList
 	args := []string{
 		"--options", "pv_all,vg_name",
 		"--reportformat", "json",
 		"--units", "b",
 	}
-	output, _, err := RunCommandSplit(PVList, args...)
+
+	// Are we running privileged or not?
```

Comment:
this is duplicated, add function to do this?

Reply:
dropped this from diff anyway